source/MRMeshC/MRVoxelsLoad.cpp

```diff
@@ -0,0 +1,38 @@
+#include "MRVoxelsLoad.h"
+
+#include "detail/TypeCast.h"
+#include "detail/Vector.h"
+
+#include "MRVoxels/MRVoxelsLoad.h"
+
+using namespace MR;
+REGISTER_AUTO_CAST2( std::string, MRString )
+REGISTER_AUTO_CAST( FloatGrid )
+REGISTER_AUTO_CAST( Vector3f )
+REGISTER_AUTO_CAST( Vector3i )
+MR_VECTOR_LIKE_IMPL( VdbVolumes, VdbVolume )
+
+MRVdbVolumes* mrVoxelsLoadFromAnySupportedFormat( const char* file, MRProgressCallback cb_, MRString** errorStr )
+{
+    auto cb = [cb_] ( float progress ) -> bool { return cb_( progress ); };
+    auto res = cb_ ? VoxelsLoad::fromAnySupportedFormat( file, cb ) : VoxelsLoad::fromAnySupportedFormat( file );
+
+    if ( res )
+    {
+        auto volumes = mrVdbVolumesNewSized( res->size() );
```

Comment:
mb just new and resize? not to introduce new function for each vector?
(in general it is OK. but minimizing new API should be preferred for following auto generation simplicity)